app/views/videos/show.html.haml

```diff
@@ -2,14 +2,38 @@
   .container
     .row
       .video_large_cover.col-sm-7.col-sm-offset-1
-        %img(src="tmp/monk_large.jpg")
+        %img(src="#{@video.large_cover_url}")
       .video_info.col-sm-3
         %header
           %h3 
             = @video.title
-          %span Rating: 4.5/5.0
+          %span Rating: #{@video.average_rating}
         %p 
           = @video.description
         .actions
           %a.btn.btn-primary(href="") Watch Now
-          %a.btn.btn-default(href="") + My Queue
\ No newline at end of file
+          = link_to "+ My Queue", queue_items_path(video_id: @video.id), method: :post, class: 'btn btn-default'
+
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, Review.new] do |f|
+        %label Rate this video
+        .row
+          .col-sm-3
+            = f.select :rating, options_for_select([5,4,3,2,1].map {|number| [pluralize(number, "Star")]})
```

Comment:
What you pass to controller should be just the `number`. (`[pluralize(number, "Star"), number]`)